src/main/java/com/osiris/autoplug/client/tasks/updater/server/TaskServerUpdater.java

```diff
@@ -403,114 +300,31 @@ private void doFabricUpdatingLogic() throws WrongJsonTypeException, IOException,
         }
     }
 
-    private void doPaperUpdatingLogic() throws WrongJsonTypeException, IOException, HttpErrorException, InterruptedException, YamlWriterException, DuplicateKeyException, YamlReaderException, IllegalListException {
-
-        PaperDownloadsAPI paperDownloadsAPI = new PaperDownloadsAPI();
-        int build_id = updaterConfig.server_build_id.asInt();
-        int latest_build_id = paperDownloadsAPI.getLatestBuildId(serverSoftware, serverVersion);
-
-        // Check if the latest build-id is bigger than our current one.
-        if (latest_build_id <= build_id) {
-            setStatus("Your server is on the latest version!");
-            setSuccess(true);
-            return;
-        }
-
+    private void doMCServerUpdaterLogic() {
         if (profile.equals("NOTIFY")) {
-            setStatus("Update found (" + build_id + " -> " + latest_build_id + ")!");
-        } else if (profile.equals("MANUAL")) {
-            setStatus("Update found (" + build_id + " -> " + latest_build_id + "), started download!");
-
-            // Download the file
-            String build_hash = paperDownloadsAPI.getLatestBuildHash(serverSoftware, serverVersion, latest_build_id);
-            String build_name = paperDownloadsAPI.getLatestBuildFileName(serverSoftware, serverVersion, latest_build_id);
-
-            File cache_dest = new File(downloadsDir.getAbsolutePath() + "/" + serverSoftware + "-latest.jar");
-            if (cache_dest.exists()) cache_dest.delete();
-            cache_dest.createNewFile();
-            TaskDownload download = new TaskDownload("ServerDownloader", getManager(),
-                    paperDownloadsAPI.getDownloadUrl(serverSoftware, serverVersion, latest_build_id, build_name),
-                    cache_dest);
-            download.start();
-
-            while (true) {
-                Thread.sleep(500); // Wait until download is finished
-                if (download.isFinished()) {
-                    if (download.isSuccess()) {
-                        setStatus("Server update downloaded. Checking hash...");
-                        if (download.compareWithSHA256(build_hash)) {
-                            setStatus("Server update downloaded successfully.");
-                            setSuccess(true);
-                        } else {
-                            setStatus("Downloaded server update is broken (hash check failed). Nothing changed!");
-                            setSuccess(false);
-                        }
-
-                    } else {
-                        setStatus("Server update failed!");
-                        setSuccess(false);
-                    }
-                    break;
-                }
-            }
+            setStatus("Unsupported profile: " + profile);
+            setSuccess(false);
```

Comment:
Notify profile is supported.
It simply says that an update was found, and does nothing else.